website/content/en/docs/building-operators/ansible/development-tips.md

```diff
@@ -55,35 +55,19 @@ Sometimes it is beneficial for a developer to run the Ansible code from their
 local machine as opposed to running/rebuilding the operator each time. To do
 this, initialize a new project:
 ```bash
-$ operator-sdk new --type ansible --kind Foo --api-version foo.example.com/v1alpha1 foo-operator
-Create foo-operator/tmp/init/galaxy-init.sh
-Create foo-operator/tmp/build/Dockerfile
-Create foo-operator/tmp/build/test-framework/Dockerfile
-Create foo-operator/tmp/build/go-test.sh
-Rendering Ansible Galaxy role [foo-operator/roles/Foo]...
-Cleaning up foo-operator/tmp/init
-Create foo-operator/watches.yaml
-Create foo-operator/requirements.yml
-Create foo-operator/deploy/rbac.yaml
-Create foo-operator/deploy/crd.yaml
-Create foo-operator/deploy/cr.yaml
-Create foo-operator/deploy/operator.yaml
-Run git init ...
-Initialized empty Git repository in /home/dymurray/go/src/github.com/dymurray/opsdk/foo-operator/.git/
-Run git init done
-
-$ cd foo-operator
+$ mkdir foo-operator && cd foo-operator
+$ operator-sdk init --plugins=ansible --domain=example.com --group=foo --version=v1alpha1 --kind=Foo
```

Comment:
It will not gen the roles. The roles dir will be empty by default. We need to use the `--generate-role` for that now.

Reply:
Updated